app/frontend/src/features/communities/hooks.ts

```diff
@@ -160,6 +167,78 @@ export const useNewDiscussionMutation = (onSuccess?: () => void) => {
   );
 };
 
+export const useListAvailableReferences = (
+  userId: number,
+  type: "received" | "given" | "all"
+) =>
+  useQuery<AvailableWriteReferencesRes.AsObject, GrpcError>(
+    referencesKey(userId, type),
```

Comment:
I should have removed this query key... but it doesn't really make sense to use here, since this is:
- not a resource about a reference itself
- the `availableWriteReferences` RPC returns all references you can write for that particular user, so it doesn't make sense to scope this to a specific "type"

```suggestion
    availableWriteReferencesKey(userId),
```
I suggest doing something like this to mirror the language used in the backend/protobuf contract already. Then in `queryKeys`, export this as 
```typescript
const availableWriteReferencesKey = (userId: number) => ["availableWriteReferences", { userId }]
```
or something like that